challenges.yaml

```diff
@@ -126,7 +126,7 @@
       Extract all IP addreses from files that
       that start with "access.log" printing one
       IP address per line.
-    example: find . -name "access.log*" -exec grep -Po '^\d+\.\d+\.\d+\.\d+' {} \;
+    example: grep -Po '^\d+\.\d+\.\d+\.\d+' **/access.log*
```

Comment:
This has a different output than before because of the path prefix. Maybe I'll do a regex to make it optional.

```
extract_ip_addresses	`grep -Po '^\d+\.\d+\.\d+\.\d+' **/access.log*`	✓	✘
var/log/httpd/access.log:163.56.115.58
var/log/httpd/access.log:75.113.188.234
...
```